KerbalStuff/profiling.py

```diff
@@ -0,0 +1,15 @@
+import random
+from typing import TYPE_CHECKING
+
+from .config import _cfgi
+
+if TYPE_CHECKING:
+    from wsgiref.types import StartResponse, WSGIEnvironment
+
+def sampling_function(environ: "WSGIEnvironment") -> bool:
+    # Don't bother profiling admin pages or static files
+    if any(filter(environ.get('PATH_INFO', '').startswith,
+                  [ "/admin", "/content", "/static" ])):
```

Comment:
```suggestion
    if environ.get('PATH_INFO', '').startswith(("/admin", "/content", "/static")):
```
Feels a bit cleaner and might be slightly more performant, I fear `filter()`  re-executes `environ.get()` each time.

Reply:
I didn't know we could use a tuple here!

Reply:
This and the search done in the latest force push.